src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/RemoveMock.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.companieshouse.overseasentitiesapi.mocks;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
+
+public class RemoveMock {
+    public static RemoveDto getRemoveDto() {
+        RemoveDto removeDto = new RemoveDto();
```

Comment:
you can just return the new RemoveDto() here
instead of 
`RemoveDto removeDto = new RemoveDto();
return removeDto;
`

you can just use :
`return new RemoveDto();`

Reply:
return statement added